src/app/pages/HomePage/ParaTimeSelector/Graph/graph-utils.ts

```diff
@@ -1,7 +1,14 @@
 import { ScaleToOptions } from 'react-quick-pinch-zoom'
 import { GraphEndpoint, GraphEndpoints } from './types'
+import { ParaTime } from '../../../../../config'
 
 export abstract class GraphUtils {
+  private static ENABLED_GRAPH_ENDPOINTS: GraphEndpoint[] = [ParaTime.Emerald]
```

Comment:
The reason for it is that Consensus wasn't included in `ParaTime` enum. This will be consolidated in https://github.com/oasisprotocol/explorer/blob/60b42f0c35328c6e1bf19f82351c343156304867/src/app/pages/HomePage/ParaTimeSelector/Graph/types.ts